canonical/ubuntu.com
# WD-12033 - Update the Suru across the credentials bubble

## Done

- Update the Suru across the credentials bubble
- Small improvements across the content

## QA

- Open the demo
- Navigation around the /credentials bubble using the files changed
- See how they look logged in and out

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12033
